maya/mel/assign-pbr-textures.mel

```diff
@@ -0,0 +1,43 @@
+{
+    $sl = {""};
+    $sl = `ls -sl`;
+    
+    if($sl[0]=="")
+    {
+        print "No selection to apply the material to...\n";
+    }
+    else
+    {
+        loadPlugin "C:/Program Files/Autodesk/Maya2017/bin/plug-ins/glslShader.mll";
+        source "createAndAssignShader.mel";
+    
+        $sh = `shadingNode -asShader GLSLShader`;
+        setAttr -type "string" ($sh+".shader") "D:\\Development\\PBR\\shaders\\glTF_PBR.ogsfx";
```

Comment:
The path to the shader should not be hardcoded, best to pass this as a required argument to the procedure.